### ISSUE TITLE
The select component doesn't announce its selected option in VoiceOver

### ISSUE THREAD

**Issue by stowball** [dequelabs/pattern-library#21]
When moving through the options with the keyboard in Chrome, the selected item is not announced in VoiceOver

**Comment by schne324** [comment]
This is a classic example where we need some extensive AT testing before we can decide on a solution here.  As the select component exists today, we are using `aria-activedescendant` along with `aria-selected` which _should_ work according to the spec but unfortunately there is a known webkit bug (https://github.com/dequelabs/pattern-library/blob/01ceb17a5365a35490d5d3d28ee6af736d15488c/lib/components/selects/arrow.js#L34) which deals with support for `aria-activedescendant`. As a result, we've added in _some_ useful live region announcements (but now we're stuck with a weird double up where activedescendant is supported). As a lot of these types of AT-lacking-support issues go, we will likely rely solely on live regions to convey stuff like selected option and remove `aria-selected`/`aria-activedescendant` completely from the code for now (until Safari+VO decide to support these attributes).

**Comment by stowball** [comment]
Interestingly, alphagov make it work somehow https://github.com/alphagov/accessible-autocomplete

**Comment by philsherry** [comment]
Switching this to use a native controls (same as in #65), might help with such bugs. 

Seeing as you've mentioned the GOV.UK version, does yours meet the same [Accessibility Acceptance Criteria](https://github.com/alphagov/accessible-autocomplete/blob/master/accessibility-criteria.md)?

**Comment by schne324** [comment]
This has been fixed in pattern-library@5.0.0 still haven't deployed to production but you can see the fix here: https://pattern-library-test.dequelabs.com/components/selects
